### PR TITLE
Bug 2017535: fix broken Argo CD link image

### PR DESCRIPTION
**Pull request by reginapizza**
Description: In the GitOps Environments Details page, the image before the link to Argo CD UI was showing a broken image because the Argo CD operator (the source it was grabbing it from) was not available in the cluster. I have changed it to be a link to the argocd logo in the argocd project, which is unlikely to change. 

Jira Id : https://issues.redhat.com/browse/GITOPS-1500

Screen shots:
non-broken-argocd-link-image

### CODE REVIEW

File: frontend/packages/gitops-plugin/src/components/details/GitOpsDetails.tsx [hunk 1]
```diff
@@ -179,7 +176,7 @@ const GitOpsDetails: React.FC<GitOpsDetailsProps> = ({ envs, appName }) => {
                                   <span className="odc-gitops-details__env-section__argo-external-link">
                                     <img
                                       loading="lazy"
-                                      src={ArgoCDIcon}
+                                      src="https://cd.apps.argoproj.io/assets/images/logo.png"
```
Comment (andrewballantyne) [inline frontend/packages/gitops-plugin/src/components/details/GitOpsDetails.tsx:179]: As I mentioned in a DM to Regina; if this is not available in offline envs, we probably should look to get another icon.

**Comment by reginapizza** [comment]
Ok I've fixed this so it now has a locally stored PNG file. This is what it looks like in console now: 

argoIcon 

(also removed a commented out import I saw)

**Review by andrewballantyne — APPROVED** [review]

**Pull request merged**